presto-docs/src/main/sphinx/clients/presto-cli.rst

```diff
@@ -177,7 +192,8 @@ OPTIONS
             key=value)
 
         --runtime-stats
-            Enable runtime stats information. Flag must be used in conjunction with the --debug flag
+            Enable runtime stats information. Flag must be used in conjunction
+            with the --debug flag
```

Comment:
```suggestion
            with ``--debug``
```

Reply:
Hmm, the output here is just copy-and-paste directly from the terminal into the doc. I'm not sure if making additional modifications will be fruitful as if we need to update it in the future the changes might get lost.

Is this flag not clearly readable without the backticks? I haven't actually checked the rendering in my browser

Reply:
Good point! Yeah, forget this nit, it's fine.